_layouts/splash.md

```diff
@@ -32,8 +32,8 @@ layout: default
               </div>
             </div>
             <div class="mx-1 col">
-              <a href="/gmui" class="text-secondary">
-                <i class="fas fa-users service-link" data-toggle="tooltip" data-placement="top" title="Manage your CANFAR groups"></i>
+              <a href="" id="gmui_link" class="text-secondary">
+                <i class="fas fa-users service-link" data-toggle="tooltip" data-placement="top" title="Manage your CADC groups"></i>
```

Comment:
The documentation below states that this will manage the "CADC/CANFAR" groups, but this title says only CADC.  We should change one of them.

Also, I think it would be good if it opened in a `_blank` target.

Reply:
It seems to open in a separate tab in both Chrome and Firefox. Not certain why.  I can make it explicit, though.